pom.xml

```diff
@@ -70,7 +70,7 @@
         <lombok.version>1.18.38</lombok.version>
 
         <!-- temporary version waiting for a real automatic release in creedengo repository -->
-        <creedengo-rules-specifications.version>2.2.2</creedengo-rules-specifications.version>
+        <creedengo-rules-specifications.version>2.4.0</creedengo-rules-specifications.version>
```

Comment:
put the "main-SNAPSHOT" until the next release (yo can install locally your rules-spec in "main-SNAPSHOT" verison and check your IT)